js/algoliasearch/instantsearch.js

```diff
@@ -75,6 +75,13 @@ document.addEventListener("DOMContentLoaded", function (event) {
 				if (algoliaConfig.request.refinementKey.length > 0) {
 					data.helper.toggleRefine(algoliaConfig.request.refinementKey, algoliaConfig.request.refinementValue);
 				}
+				
+				if (algoliaConfig.areCategoriesInFacets === false && algoliaConfig.request.path.length > 0) {
```

Comment:
I need to set the refinement only when categories widget does not exists and the user is on category page. Otherwise it breaks down categories hierarchical widget.